pkg/auth/oauth/handlers/default_auth_handler.go

```diff
@@ -44,6 +45,8 @@ const (
 	warningHeaderAgentIndex = 2
 	warningHeaderTextIndex  = 3
 	warningHeaderDateIndex  = 4
+
+	useRedirectParam = "useRedirectHandler"
```

Comment:
I wouldn't feel bad about changing this later, but if we're going to expose it, let's at least pick a value we're reasonably happy with out of the gate. Maybe `idp`?